modules/core/client/views/enroll.client.view.html

```diff
@@ -1,4 +1,5 @@
-<section data-ng-controller="EnrollController">
+<section data-ng-controller="EnrollController" id="content">
```

Comment:
It's the same side bar, it's just being called through a directive. I tried putting it into the index file, but I couldn't find a way for it to not appear on the login screen.